docs/faq/users/transfer-files-on-rm.rst

```diff
@@ -0,0 +1,43 @@
+Example: Transfer a job's output sandbox on removal
+===================================================
+
+When a user :tool:`condor_rm`'s a job, HTCondor assumes the job is
+not needed anymore, and removes the scratch directory, and never
+transfers any files from the sandbox back to the AP, or wherever
+they would be transferred to upon completion.
+
+There is a job classd attribute to disable this, so that any
+intermediate files that exist in the sandbox will be transferred,
+in the same way that they would have been if user had run
+:tool:`condor_vacate_job`. This attribute is named *MY.SpoolOnEvict*.
```

Comment:
```suggestion
There is a job ClassAd attribute to disable this, so that any
intermediate files that exist in the sandbox will be transferred,
in the same way that they would have been if user had run
:tool:`condor_vacate_job`. This attribute is :ad-attr:`My.SpoolOnEvict`.
```